code/game/objects/items/dice.dm

```diff
@@ -5,15 +5,26 @@
 	desc = "Contains all the luck you'll ever need."
 	icon = 'icons/obj/dice.dmi'
 	icon_state = "dicebag"
+	var/list/special_die = list(
+				/obj/item/dice/d1 = 1,
+				/obj/item/dice/d2 = 1,
+				/obj/item/dice/fudge = 1,
+				/obj/item/dice/d6/space = 1,
+				/obj/item/dice/d00 = 1,
+				/obj/item/dice/eightbd20 = 1,
+				/obj/item/dice/fourdd6 = 1,
+				/obj/item/dice/d100 = 1
+				)

```

Comment:
I believe this doesn't need to be an assoc list, try removing the `= 1` and seeing if that still works.

Reply:
You're right, the lootspawners are weigted and this is no longer.